score/application.py

```diff
@@ -0,0 +1,56 @@
+from config import settings
+import click
+from pathlib import Path
+import logging
+
+from etl.sources.census.etl_utils import reset_data_directories as census_reset
```

Comment:
We should have a conversation about this in the styleguide, I generally prefer using the full path to modules rather than importing them. 

E.g., `import etl` and then 

```
etl.sources.census.etl_utils.reset_data_directories(...)
```

It's more verbose, but more explicit and makes it more obvious to the reader where everything's coming from. Plus you don't have to do the renamings as you've done here. :)

Reply:
There is no style recommendations on imports that I could find, [only this recommendation about breaking long ones](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses).

However, readability is one of the most important recommendations, I would find difficult to read long module names within the code -- you would hit the 80 character line length easily and would need to use `\` characters across the code, so I am not in favor, but curious to hear what others think.

Reply:
Both added to tracking ticket https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/255